### DISCUSSION TITLE
✈ Add support for WooCommerce endpoints in the mega Menu current-page indicator (Migrated)

### DISCUSSION THREAD

**Discussion by 79ho3ein** [elementor/elementor#31452] [Feature Request]
### Prerequisites

- [ ] I have searched for similar issues in open and closed tickets and cannot find a duplicate.
- [ ] I have troubleshooted my issue, and it still exists against the latest stable version of Elementor.

### Description

Hi elementor team,

On the my account page, I use the menu element as navigation. But the problem is that this element does not recognize the active mode of WooCommerce endpoints.
For example, one of the menu options is a link to the WooCommerce order page. But if we are on the orders page, the style of the active status in the menu element does not work.

### Steps to reproduce

1- On the my account page, add the menu element. In this element, add an menu item for orders and put the WooCommerce order link there.

2- In the menu element, customize the active status according to your taste.

3- View my account in the front end and enter the orders page.

You will see that the active state does not work for the menu element. It actually only has problems with WooCommerce endpoints.

**If you use the WordPress menu element to display your WoCommerce endpoints, everything works as expected.**

### Agreement

- [X] I agree that my issue may be closed without action if it doesn't meet all the requirements.

**Comment by Tafmakura** [comment]
One way I solved this was to make sure that when you add the url to the menu item you add the full address including https:// etc. Shortened URLs will not work

**Comment by nicholaszein** [comment]
Hi, @79ho3ein ,

Thank you for reaching out! 🙏 

### ⛔ After reviewing your report we concluded this is not a bug caused by Elementor. The feature you're referring to is not yet supported.

I am migrating this issue to our Feature Requests Discussion.

Kind regards
